### ISSUE TITLE
Bootstrap version?

### ISSUE THREAD

**Issue by docwilmot** [backdrop-contrib/fanciblock#31]
Working on Bootstrap 5 Lite and realized this module would work great for integrating additional functionality into that theme. Would also be useful even with other themes since layouts are built with Bootstrap.

**Comment by stpaultim** [comment]
@docwilmot - Can you clarify your point?

Are you suggesting that the functionality of fanciblock should be built into some of these themes or simply recommending that they might work well together?

**Comment by docwilmot** [comment]
I dont think it was entirely clear to me either. Point is the components this module produces overlaps with some of the things that Bootstrap does, like [heros, features, jumbotron](https://getbootstrap.com/docs/5.0/examples/), [cards](https://getbootstrap.com/docs/5.0/components/card/), etc.

So I'm thinking either use Bootstrsp's markup in this module (because Bootstrap does it better and has more options), or create a similar module *for* Bootstrap Lite, (a Bootstrsp Tools Module?) that allows users to create all these Bootstrap components.

**Comment by stpaultim** [comment]
Maybe something like this? https://backdropcms.org/project/views_bootstrap
